File: Asklepios/Asklepios.Core/Entities/Examinations/ExamResult.cs

using Asklepios.Core.Entities.Patients;

namespace Asklepios.Core.Entities.Examinations;

public class ExamResult
{
    public Guid ExamResultId { get; set; }
    
    public Guid PatientId { get; set; }
    public Patient Patient { get; set; }
    
    public DateOnly Date { get; set; }
    public string Result { get; set; }
    public string Comment { get; set; }
    
    public int ExaminationId { get; set; }
    public Examination Examination { get; set; }
}

File: Asklepios.Web/src/models/Examinations/examResult.ts
export interface ExamResultDto {
  examResultId: string;
  patientId: string;
  date: Date;
  result: string;
  comment: string;
  examId: number;
 }

 export interface ExamResultListDto {
  examResultId: string;
  patientId: string;
  date: Date;
  result: string;
  comment: string;
  examName: string;
 }

 export class InputCreateExamResult {
  examResultId:string = '';
  patientId:string = '';
  date: Date = new Date();
  result:string = '';
  comment:string = '';
  examId: number | null = null;
 }
